File: android/app/src/main/java/abi10_0_0/host/exp/exponent/ReactUnthemedRootView.java
// Copyright 2015-present 650 Industries. All rights reserved.

package abi10_0_0.host.exp.exponent;

import android.content.Context;
import android.view.ContextThemeWrapper;

import abi10_0_0.com.facebook.react.ReactRootView;
import host.exp.expoview.R;

public class ReactUnthemedRootView extends ReactRootView {
  public ReactUnthemedRootView(Context context) {
    super(new ContextThemeWrapper(
        context,
        R.style.Theme_Exponent_None
    ));
  }
}

File: android/versioned-abis/expoview-abi49_0_0/src/main/java/abi49_0_0/host/exp/exponent/ReactUnthemedRootView.kt
// Copyright 2015-present 650 Industries. All rights reserved.
package abi49_0_0.host.exp.exponent

import android.content.Context
import android.view.ContextThemeWrapper
import host.exp.expoview.R
import abi49_0_0.com.facebook.react.ReactRootView

class ReactUnthemedRootView(context: Context?) : ReactRootView(
  ContextThemeWrapper(
    context,
    R.style.Theme_Exponent_None
  )
)
